CHANGELOG.md

```diff
@@ -6,12 +6,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 ### Changed
+* New CHANGELOG format (@thomasfedb)
```

Comment:
Please remove this line as it's not a change to CarrierWave itself per se, rather a change to how we document changes.